Cargo.toml

```diff
@@ -2,6 +2,12 @@
 members = [
   "crates/stdsimd-verify",
   "crates/stdsimd",
+  "crates/coresimd_v16_tests",
```

Comment:
Could these perhaps be organized under `crates/coresimd/tests/*.rs`?

That'd mean that `cargo test -p coresimd` should hopefully pick up everything